src/opera/parser/tosca/v_1_3/definition_collector_mixin.py

```diff
@@ -41,6 +43,20 @@ def collect_interface_definitions(self, service_ast):
                 defs[name]["operations"].update(definition["operations"])
 
         for name, definition in self.get("interfaces", {}).items():
+            if name == StandardInterfaceOperation.shorthand_name() or name == StandardInterfaceOperation.type_uri():
+                valid_standard_interface_operation_names = [i.value for i in StandardInterfaceOperation]
+                for operation in definition.get("operations", {}):
+                    if operation not in valid_standard_interface_operation_names:
+                        self.abort("Invalid operation for {} interface: {}. Valid operation names are: {}"
+                                   .format(name, operation, valid_standard_interface_operation_names), self.loc)
```

Comment:
I did not do that because these are two special TOSCA cases because TOSCA defines just these two main interface types so I believe that making a function here would not be so beneficial as it cannot be reused for any other cases.

Reply:
Okay!